### ISSUE TITLE
Add warning when using plugins with `.esl` extension in VR games

### ISSUE THREAD

**Issue by Pickysaurus** [loot/loot.github.io#110]
Suggestion: Generate a LOOT error message if the user has included an ESL or light flagged ESP in their Skyrim VR load order. 

Not sure if this can be done at a MasterList level, so feel free to move this. 

https://forums.nexusmods.com/index.php?/topic/11472988-vr-lack-of-support-for-esl-files/

**Comment by Ortham** [comment]
> From some cursory research, ESL-flagged ESPs don't seem to cause a problem (the flag is just ignored), it only appears that plugins with the `ESL` file extension are ignored. If anyone knows of any authoritative sources, that'd be helpful. I don't have Skyrim VR to test, so I'm not sure how exactly LOOT handles ESL plugins in it (@Ortham?) or how other mod managers do for that matter.
> 
> If we don't already, we probably shouldn't display plugins with the `ESL` extension in Skyrim VR (FO4 VR too). We could then add a global warning message if any such plugins are found with a simple regex. There's also the issue of plugins that have an ESL as a master, we might need a specific message for that case as well. That'd require something on the app's end.

I don't think LOOT handles ESL-flagged plugins in Skyrim VR any differently from Skyrim SE.

**Comment by sibir-ine** [comment]
> > From some cursory research, ESL-flagged ESPs don't seem to cause a problem (the flag is just ignored), it only appears that plugins with the `ESL` file extension are ignored. If anyone knows of any authoritative sources, that'd be helpful. I don't have Skyrim VR to test, so I'm not sure how exactly LOOT handles ESL plugins in it (@Ortham?) or how other mod managers do for that matter.
> > If we don't already, we probably shouldn't display plugins with the `ESL` extension in Skyrim VR (FO4 VR too). We could then add a global warning message if any such plugins are found with a simple regex. There's also the issue of plugins that have an ESL as a master, we might need a specific message for that case as well. That'd require something on the app's end.
> 
> I don't think LOOT handles ESL-flagged plugins in Skyrim VR any differently from Skyrim SE.

It'd probably be a good idea to change that behavior for the VR games then. Though, we might want to wait for @pStyl3 to test it when they have the chance.

**Comment by pStyl3** [comment]
I've ordered the Valve Index now, so hopefully in around 2 to 3 weeks I have things set up.

**Comment by pStyl3** [comment]
The Valve Index has been delivered and I've spend a bit of the day setting everything up. This included:
- Installing SteamVR
- Placing two base stations and powering them
- Charging the empty batteries of the two controllers
- Connecting the headset itself (power, USB3.0 and Displayport connectors are needed)
- Installing software updates for all five devices individually (heatset, 2 base stations, 2 controllers)

Once everything was ready, I've started a new game in SkyrimVR and played until I've reached Whiterun, as well as Fallout 4 VR, which I've played until I reached Sanctuary. So now I am ready to test things for this issue.
